net.crossfireatlas.crossfire.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
```

Comment:
You need to explicitly reference this file in your manifest and install it at correct place.

Reply:
OK, I have added installing .desktop and .appdata.